ci/jenkins/test-vm.sh

```diff
@@ -274,6 +274,7 @@ function run_e2e_vms {
 
     configure_vm_agent
     echo "====== Running Antrea e2e Tests for VM ======"
+    set +e
```

Comment:
Why needs this? Isn't `+e` the default value?

Reply:
@Nithish555 do you remember why we added this?

Reply:
Because if the error is thrown if we want to ignore and continue the script.